### ISSUE TITLE
A11y_Model Builder_azure_Create New Experiment_Usability: The "New Subscription, New workspace, New Compute" control is not clearly visible it is getting truncated.

### ISSUE THREAD

**Issue by Harika1702** [dotnet/machinelearning-modelbuilder#2654]
### GitHub Tags
#A11ySev4; #A11yTCS; #A11yUsable; #DesktopApp; #Win11; #Visual Studio Client; #SH_ModelBuilder_Win32_May2023; #FTP; #Win32; #Benchmark;

### Environment
Application Name: Model Builder
Microsoft Visual Studio Enterprise 2022 (64-bit) - Current
Version 17.6.1
Microsoft Windows 11 Enterprise Build 22624.1755

### Repro Steps

1. Launch the Visual Studio with at least one of the .NET workloads.
2. Tab till enable ML.NET Model Builder (Tools -> Options -> Environment -> Preview Features) and select it.
3. Create a new .NET project or open an existing .NET project.
4. Open context menu (Right-click) of the project in the solution explorer.
5. Tab till Add and then tab till Machine Learning Model and press enter.
6. Model Builder dialog will open. Tab till Scenario tab.
7. Select Scenario screen will appear.
8. Tab till "Image Classification" and press enter.
9. Setup environment screen will appear, Tab till Azure tile and select it.
10. Tab till Setup workspace and press enter.
11. Now tab till "New Subscription" Control.
12. Now verify that The "New Subscription, New workspace, New Compute" control is not clearly visible it is getting truncated.

### Actual Result
The "New Subscription, New workspace, New Compute" controls is not clearly visible it is getting truncated.
**Note: we are facing similar issue throughout the application.**

### Expected Result
The "New Subscription, New workspace, New Compute" controls should be clearly visible without any truncations/overlapping or it should have tooltip.

### User Impact
Low visibility users will not be able to know the control if the control is getting truncated.

### Attachment
![Bug_Usability](https://github.com/dotnet/machinelearning-modelbuilder/assets/100270074/0d7f1b70-6bcb-4c02-ab9b-d3a1780681f1)

**Comment by zewditu** [comment]
@Harika1702 ,this  can be validated by a private build you can download from [here](https://microsoft.sharepoint.com/sites/PrivateBuildforAccessibility/Shared%20Documents/Forms/AllItems.aspx?id=%2Fsites%2FPrivateBuildforAccessibility%2FShared%20Documents%2FMicrosoft%2EML%2EModelBuilder%20%2821%29%2Evsix&parent=%2Fsites%2FPrivateBuildforAccessibility%2FShared%20Documents
).

**Comment by Samhitha66** [comment]
@zewditu
We don't have access for the link which you provided in the above comment.
Please provide us the access. So that we will verify this bug.
#Regressed:07-13-23; #PendingEnvironment;
![image](https://github.com/dotnet/machinelearning-modelbuilder/assets/99243079/99e3f9b6-3d9f-4125-bf56-771fecbba275)

**Comment by Samhitha66** [comment]
@zewditu
Regressed the issue in the below environment
Microsoft Visual Studio Enterprise 2022 (64-bit) - Preview
Version 17.8.0 Preview 1.0 [33912.440.main]
The issue is fixed. Hence closing the bug.
#Regressed:07-14-23; #Closed;

![image](https://github.com/dotnet/machinelearning-modelbuilder/assets/99243079/87b9241b-5270-4488-b724-ffce854d4320)
